src/main/java/com/google/javascript/clutz/ImportRenameMapBuilder.java

```diff
@@ -185,6 +204,36 @@ private static String getGoogModuleId(Node astRoot) {
             importRenameMap.put(localSymbolName, exportedSymbolName);
           }
         }
+      } else if (isVariableDestructuringAssignment(statement)) {
+        // `const B = goog.require()`
+        // `const {C, Clazz: RenamedClazz} = B`
+        // On separate lines
+        String destructuredVariable = statement.getFirstChild().getChildAtIndex(1).getString();
+
+        for (Node destructured : statement.getFirstFirstChild().children()) {
+          String originalName = destructured.getString();
+          // Destructuring can use the original name `const {A} = goog.require("foo.a")` or rename
+          // it `const {A: RenamedA} = ...`, and closure uses whichever in the symbol name it
+          // generates, so we have to extract it.
+          String variableName;
+          if (destructured.getFirstChild() != null) {
+            variableName = destructured.getFirstChild().getString();
```

Comment:
because of closures' messy AST, it is not clear which branch is the renaming destructuring. Add a one line comment.

Reply:
Done.